StrategyDesignPattern/src/Vehicle.java

```diff
@@ -0,0 +1,11 @@
+public class Vehicle {
+    DriverStrategy driverStrategy;
+
+    Vehicle(DriverStrategy driverStrategy) {
+        this.driverStrategy = driverStrategy;
+    }
+
+    public void drive() {
+        driverStrategy.drive();
+    }
+}
```

Comment:
let's make this class private,
It will allow 

Vehicle sportyVehicle = new Vehicle(new SpecialDriveStrategy());